### PR TITLE
Parse POWER_SUPPLY_CAPACITY

### PR DESCRIPTION
**Pull request by jpalus**
If POWER_SUPPLY_{CHARGE,ENERGY}_NOW is missing then try to use
POWER_SUPPLY_CAPACITY to determine current charge level.

### CODE REVIEW

File: linux/Battery.c [hunk 1]
```diff
@@ -192,6 +192,8 @@ static void Battery_getSysData(double* level, ACPresence* isOnAC) {
 
    unsigned long int totalFull = 0;
    unsigned long int totalRemain = 0;
+   bool now = false;
```
Comment (cgzones) [inline linux/Battery.c:195]: This scope change does change the logic, as now `now` does not get reset to `false` for the second and following batteries.

Comment (jpalus) [reply linux/Battery.c]: Ouch you're right, I got confused by all the scopes around. Updated however now it's limited to case when full is available while now is not (which I'm affected by).

**Review by cgzones — APPROVED** [review]
LGTM

**Comment by cgzones** [comment]
Thanks, applied.

**Pull request merged**